paths.yaml

```diff
@@ -47,6 +47,18 @@
     "$ref": "./paths/styleguides/update.yaml"
   delete:
     "$ref": "./paths/styleguides/destroy.yaml"
+"/accounts/{account_id}/custom_metadata/labels":
```

Comment:
so we decided to leave `labels` in the API rather than renaming to `properties`?

Reply:
so now needs to be updated here too, right?